practico_08/autogestion_alumnos/templates/base.html

```diff
@@ -15,5 +15,7 @@
         {% include 'navbar.html' %}
     {% endblock %}
 
-    {% block content %}{% endblock %}
+    {% block content %}
+    {% endblock %}
+    {% include 'footer.html' %}
```

Comment:
I think this is wrong, the include shouldn't be between block and endblock? if not we should delete these two first lines.